src/main/java/seedu/contax/logic/commands/DeleteTagCommand.java

```diff
@@ -0,0 +1,60 @@
+package seedu.contax.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.contax.commons.core.Messages;
+import seedu.contax.commons.core.index.Index;
+import seedu.contax.logic.commands.exceptions.CommandException;
+import seedu.contax.model.Model;
+import seedu.contax.model.tag.Tag;
+
+/**
+ * Deletes a tag identified using it's displayed index from the address book.
+ */
+public class DeleteTagCommand extends Command {
+    public static final String COMMAND_WORD = "deletetag";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + " : Deletes the tag identified by tag name (case-insensitive).\n"
+            + "Deleting this tag will also remove contacts who have this tag.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " n/friends";
```

Comment:
the text here is wrong, should be
```suggestion
            + "Example: " + COMMAND_WORD + "1";
```
cos you delete tags by index and not by tagname